docs/Researcher/cli-reference/runai-submit.md

```diff
@@ -368,10 +370,25 @@ runai submit --job-name-prefix -i gcr.io/run-ai-demo/quickstart -g 1
 
 > External access type to interactive jobs. Options are:
 >
->  * `portforward` (deprecated)
->  * `loadbalancer`
->  * `nodeport`
->  * `external-url`
+> * `nodeport`
+> * `external-url`
+>
+> This flag now supports more than one `service-type`. Multiple service types are supported in CSV style using multiple instances of the same option and commas to separate the values for them.
+>
+> For example:
+>
+>`runai submit test-np -p team-a -i ubuntu --service-type nodeport,port=30000:7070 --service-type external-url,port=30001`
+>
+>`runai submit test-np -p team-a -i ubuntu --service-type nodeport,port=30000:7070,port=9090 --service-type external-url,port=8888`
+>
+> This flag can also be used in the standard format, which uses a command flag for all the options and can only support one service type.
+>
+> For example:
+>
+> `runai submit test-jup -p team-a -i gcr.io/run-ai-demo/jupyter-tensorboard --service-type external-url --port 8888`
+>
+> `runai submit test-np -p team-a -i ubuntu --service-type nodeport --port 30000:7070`
+>
```

Comment:
These are examples from the "standard format. I changed "parameter" to "sevrvice type", and the examples above this show `--serbice-type external-url`.